docs/pages/versions/unversioned/sdk/permissions.md

```diff
@@ -207,32 +286,3 @@ The permissions type for changing brightness of the screen
 ### `Permissions.MOTION`
 
 The permission for accessing `DeviceMotion` and `DeviceOrientation` in the web browser. This can only be requested from a website using HTTPS (`expo web --https`). This permission cannot be silently retrieved, you can only request it. This permission can only be requested with a user interaction i.e. a button press.
-
-## Android: permissions equivalents inside `app.json`
```

Comment:
Let's add these under each `Permissions.<TYPE>` instead of a table. The `WRITE_CONTACTS` are optional for example, it's hard to document that in a simple table.